app/views/employment/CheckYourBenefitsView.scala.html

```diff
@@ -201,7 +201,7 @@ <h2 class="govuk-label--m">@messages("checkYourBenefits.reimbursedHeader")</h2>
     }
 
     @if(benefits.assetsDetailsPopulated){
-        <h2 class="govuk-label--m">@messages("checkYourBenefits.assetsHeader")</h2>
+        <h2 class="govuk-label--m">@messages("checkYourBenefits.assetsSubheading")</h2>
```

Comment:
Do we know if the spacing changes should be on the in year view as well? (the govuk-!-margin-top-9 stuff)

Reply:
That is a good point. I just verified  with Laura. I need to change that.